schema/v1.1.0/metadata-docs/Tomogram.md

```diff
@@ -455,6 +489,10 @@ attributes:
     required: true
     inlined: true
     inlined_as_list: true
+    pattern: (^[ ]*\{[a-zA-Z0-9_-]+\}[ ]*$)|(^SART$)|(^FOURIER SPACE$)|(^SIRT$)|(^WBP$)|(^UNKNOWN$)
```

Comment:
Did we get agreement on WBP as short-hand for Weighted Back Projection? It seems like a pretty hard-to-understand acronym unless we include a glossary somewhere to explain what this means.

Reply:
The `WBP` was recommended by @uermel . But, I agree that we should add a glossary to explain all the acronyms for tomogram reconstruction_method. 

cc: @junxini  @dgmccart